easybuild/framework/easyblock.py

```diff
@@ -865,12 +865,13 @@ def make_devel_module(self, create_in_builddir=False):
         # cleanup: unload fake module, remove fake module dir
         self.clean_up_fake_module(fake_mod_data)
 
-    def make_module_dep(self):
+    def make_module_dep(self, swap_info=None):
         """
         Make the dependencies for the module file.
         """
         deps = []
         mns = ActiveMNS()
+        swap_info = swap_info or {}
```

Comment:
no, mutable values as default value are of the devil, see http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments